src/ctia/stores/sql/judgement.clj

```diff
@@ -24,11 +24,11 @@
               (k/table :judgement_indicator)
               (k/database @db))))
 
-(defn realize-judgements [login new-judgements]
-  (for [new-judgement new-judgements]
-    (judgement-schema/realize-judgement new-judgement
-                                        (str "judgement-" (UUID/randomUUID))
-                                        login)))
+;; (defn realize-judgements [login new-judgements]
+;;   (for [new-judgement new-judgements]
+;;     (realize-judgement new-judgement
+;;                        (str "judgement-" (UUID/randomUUID))
+;;                        login)))
```

Comment:
I believe we should remove the commented code here and also the first commented require.